recipes/mod_wsgi/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "mod_wsgi" %}
+{% set version = "4.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 64a6a177946b78bb1eb8ecb31fb3aea5f6d306c4eccd247433563ed6afceafe5
+
+build:
+  skip: true  # [py27]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - mod_wsgi
+
+about:
+  home: https://modwsgi.readthedocs.io/en/develop/ 
+  license: "Apache 2.0"
```

SPDX identifiers use a dash here:
```suggestion
  license: Apache-2.0
```